payment_api/controller/payment_methods/handler.go

```diff
@@ -0,0 +1,241 @@
+package paymentmethods
+
+import (
+	"encoding/json" // "encoding/json" // package to encode and decode the json into struct and vice versa
+
+	"github.com/gorilla/mux" // used to get the params from the route
+	"log"
+	"net/http" // used to access the request and response object of the api
+
+	"github.com/jpgsaraceni/mvp-backend/payment_api/models"
+	pm_service "github.com/jpgsaraceni/mvp-backend/payment_api/services/payment_methods"
+)
+
+func CreatePaymentMethod(w http.ResponseWriter, r *http.Request) {
+	// set the header to content type x-www-form-urlencoded
+	// Allow all origin to handle cors issue
+	w.Header().Set("Context-Type", "application/x-www-form-urlencoded")
+	w.Header().Set("Access-Control-Allow-Origin", "*")
+	w.Header().Set("Access-Control-Allow-Methods", "POST")
+	w.Header().Set("Access-Control-Allow-Headers", "Content-Type")
+
+	// create an empty payment method of type models.PaymentMethod
```

Comment:
É padrão em Go colocar o comentário começando pelo nome da função, variável, tipo etc a que ele se refere e.g.:

```suggestion
	// paymentMethod creates an empty payment method of type models.PaymentMethod
```